README.md

```diff
@@ -181,6 +195,18 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Make a PR on any of these: [The Hasura GraphQL Engine](https://github.com/hasura/graphql-engine) and/or [The GraphQL Tutorial Series](https://github.com/hasura/learn-graphql)
 - **Notes**: See the [blog post](https://blog.hasura.io/hasura-joins-hacktoberfest-2019/) for more information and keep updated at Hasura [twitter](https://twitter.com/hasurahq)
 
+#### **QMK**
+
+- **Swag**: Shirt, Keyboard
+- **Requirements**: Interact 4 times with a QMK repo in any of these ways:
+  - Review/suggest significant changes on a PR that's labelled `hacktoberfest`
+  - Make a comment on an Issue/PR that contributes to the conversation that's labelled `hacktoberfest`
+  - Open a PR that documents any new feature that's consistent with our doc style (check out the `needs doc` label)
+  - Open a PR that translates some documentation
+  - Open a PR that fixes a bug in an issue labelled `hacktoberfest`
```

Comment:
Instead of having a list like this, I'd recommend something like "Requirements are listed on their Hacktoberfest site" with an embedded link. It keeps things cleaner. :smile: